[2023-05-05T00:15:12.411+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T00:10:00+00:00 [queued]>
[2023-05-05T00:15:12.416+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T00:10:00+00:00 [queued]>
[2023-05-05T00:15:12.416+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-05T00:15:12.416+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-05T00:15:12.416+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-05T00:15:12.426+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): delete_file> on 2023-05-05 00:10:00+00:00
[2023-05-05T00:15:12.429+0000] {standard_task_runner.py:55} INFO - Started process 349197 to run task
[2023-05-05T00:15:12.431+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'chatgpt_dag', 'delete_file', 'scheduled__2023-05-05T00:10:00+00:00', '--job-id', '4187', '--raw', '--subdir', 'DAGS_FOLDER/chatgpt_dag.py', '--cfg-path', '/tmp/tmprj9gu0k0']
[2023-05-05T00:15:12.432+0000] {standard_task_runner.py:83} INFO - Job 4187: Subtask delete_file
[2023-05-05T00:15:12.463+0000] {task_command.py:389} INFO - Running <TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T00:10:00+00:00 [running]> on host trivia-machine.us-central1-a.c.triviapractice-381517.internal
[2023-05-05T00:15:12.509+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=richc
AIRFLOW_CTX_DAG_ID=chatgpt_dag
AIRFLOW_CTX_TASK_ID=delete_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T00:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-05T00:10:00+00:00
[2023-05-05T00:15:12.514+0000] {logging_mixin.py:137} INFO - /home/rich2/airflow/scripts/chatgpt_output/seinfield_2023-05-05_00-15-08.json deleted successfully
[2023-05-05T00:15:12.514+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-05T00:15:12.516+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=chatgpt_dag, task_id=delete_file, execution_date=20230505T001000, start_date=20230505T001512, end_date=20230505T001512
[2023-05-05T00:15:12.563+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-05T00:15:12.570+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
